src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub enum Tee {
     Tdx,
     // Arm Confidential Compute Architecture
     Cca,
+    Csv,
```

Comment:
Could you please also add a comment with the meaning of the acronym, as we have above for Cca?

Reply:
Done